cypress/integration/tests/developer/reports/filter.test.ts

```diff
@@ -98,11 +101,11 @@ describe("Reports filter validations", { tags: "@tier2" }, () => {
         // Prevent hook from running, if the tag is excluded from run
         if (hasToBeSkipped("@tier2")) return;
 
-        // Delete the stakeholder
+        // Delete the applications, stakeholders, business services and tag types
+        deleteApplicationTableRows();
         deleteAllStakeholders();
         deleteAllBusinessServices();
```

Comment:
I think Business service need to be removed before applications , else it might give error .